File: repaso objetos/python/PhytonEst/Musico.py
from Persona import Persona
from Clases.Instrumento import *

class Musico(Persona):
    pass

    def tocar(self, inst):
        inst.afinar()
        inst.tocar()
        inst.tocarNota("Do")

r=Musico()

r.set__nombre__("miguel")
r.presentar()

i = Bajo()
r.tocar(i)

File: repaso objetos/java/Conciertopf/Fabrica_especial2/Musico.java
package Conciertopf.Fabrica_especial2;
import Conciertopf.Fabrica.Instrumento;
public class Musico extends Persona{
    private String p,c;
    public void tocar(Instrumento i){
        
        p =i.afinar();
        c =i.tocar();
    }
    public String getC() {
        return c;
    }
    public String getP() {
        return p;
    }
}
